OneBusAway/en.lproj/Localizable.strings

```diff
@@ -484,10 +484,10 @@
 "text_error_cant_show_shared_trip_param" = "Sorry, we can't show you that shared trip. If this continues, please let us know. Error: %@";
 
 /* Error message title displayed to the user when handoff failed to work. */
-"msg_handoff_failure_title" = "Uh oh! Handoff didn't work properly 😭";
+"msg_handoff_failure_title" = "Sorry, handoff didn't work properly";
 
 /* Error message body displayed to the user when handoff regions did not match both clients. */
-"msg_handoff_region_mismatch_body" = "Try setting your region to \"Automatically Select Region\" to fix this issue.";
+"msg_handoff_region_mismatch_body" = "Try enabling the \"Automatically Set Region\" option from the Regions section under the Info tab to fix this issue, and let us know if that doesn't work.";
```

Comment:
It is "Automatically Select Region", like you had originally. Did I say "Set"? Sorry if I did.